test/jdk/tools/jpackage/share/AddLShortcutTest.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2018, 2020, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.nio.file.Path;
+import java.io.File;
+import java.util.Map;
+import java.lang.invoke.MethodHandles;
+import jdk.jpackage.test.PackageTest;
+import jdk.jpackage.test.FileAssociations;
+import jdk.jpackage.test.AdditionalLauncher;
+import jdk.jpackage.test.JPackageCommand;
+import jdk.jpackage.test.JavaAppDesc;
+import jdk.jpackage.test.TKit;
+import jdk.jpackage.test.Annotations.Test;
+import jdk.jpackage.test.Annotations.Parameter;
+import jdk.jpackage.test.CfgFile;
+
+/**
+ * Test --add-launcher parameter with shortcuts (platform permitting).
+ * Output of the test should be AddLShortcutTest*.* installer.
+ * The output installer should provide the same functionality as the
+ * default installer (see description of the default installer in
+ * SimplePackageTest.java) plus install three extra application launchers.
+ */
+
+/*
```

Comment:
Why do you need two jtreg @test-s if there is only one test method in the test class? They are 100% duplicates.